### PR TITLE
:sparkles: beer card

### PR DESCRIPTION
**Pull request by delphinemillet**

### CODE REVIEW

File: src/components/Beers.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './Beers.css';
+import { Card } from 'antd';
+
+const Beers = () => (
```
Comment (fabienjuif) [inline src/components/Beers.jsx]: I think this is more a `Beer` than a `Beers` component

File: src/components/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export default from './Beers';
```
Comment (fabienjuif) [inline src/components/index.js]: eslint doesn't cry about that missing empty line  ?

File: src/components/Beers.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <Card className="card">
+        <img src="http://icons.iconarchive.com/icons/flat-icons.com/flat/512/Beer-icon.png" alt="beer"/>
+        <h1>Guinness</h1>
+        <div><label>%</label> 7.5</div>
+        <div><label>Color</label> Black</div>
+        <div><label>Type</label> Stout</div>
+    </Card>
+)
+
+export default Beers;
```
Comment (fabienjuif) [inline src/components/Beers.jsx]: eslint doesn't cry about that missing empty line  ?

File: src/components/Beers.css [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.card {
+    width: 200px;
```
Comment (fabienjuif) [inline src/components/Beers.css]: What about using `em` so we can scale components

File: src/components/index.js (hunk 2 above)
Comment (delphinemillet) [inline src/components/index.js]: no

**Pull request merged**